src/flaskwebgui.py

```diff
@@ -11,16 +11,26 @@
 from multiprocessing import Process
 from threading import Thread
 from dataclasses import dataclass
-from typing import Callable, Any, List, Union, Dict
+from typing import Any, Callable, Dict, List, Union
 from contextlib import suppress
 
+import psutil
 
 FLASKWEBGUI_USED_PORT = None
 FLASKWEBGUI_BROWSER_PROCESS = None
 
 OPERATING_SYSTEM = platform.system().lower()
 PY = "python3" if OPERATING_SYSTEM in ["linux", "darwin"] else "python"
 
+TEMP_DIRS_CREATED = []
+
+
+def cleanup_temp_dirs():
+    for dir_path in TEMP_DIRS_CREATED:
+        if os.path.exists(dir_path):
+            print(f'Removing Temp dir {dir_path}')
+            shutil.rmtree(dir_path, ignore_errors=True)
+
```

Comment:
No need for list and the cleanup_temp_dirs. 
On each app startup only 1 profile dir for the browser is created in tmp folder.

delete them